client/src/annoMatrix/normalize.ts

```diff
@@ -6,11 +6,16 @@ import {
   globalConfig,
 } from "../globals";
 import { Dataframe } from "../util/dataframe";
+import {
+  AnnotationColumn,
+  ColumnSchema,
+  Field,
+  Schema,
+} from "../common/types/entities";
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
 export function normalizeResponse(
-  field: string,
-  schema: any, // eslint-disable-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
+  field: Field,
+  schema: Schema,
   response: Dataframe
```

Comment:
this parameter is an actual class DataFrame instance, not a schema sub-object of the same name.  This should remain `any` until `util/dataframe` types are available.

Reply:
ditto for the return value